src/js/options.js

```diff
@@ -555,6 +558,39 @@ function removeDisabledSite(event) {
   });
 }
 
+// update the select box on the widget site allow list options page when domains are added/removed
+function reloadWidgetSitesAllowlist() {
+
+  let sites = Object.keys(OPTIONS_DATA.settings.widgetSiteAllowlist),
+    $select = $('#widget-allowlist-select');
+
+  // sort widget exemptions sites the same way other options page domains lists are
+  sites = htmlUtils.sortDomains(sites);
+
+  $select.empty();
+  for (let i of sites) {
+    $('<option>').text(i).appendTo($select);
```

Comment:
How about showing the allowed widgets in parenthesis, like in my screenshot?

Reply:
I like this idea. I do wonder if it would lead the user to desire an allow-all feature for their favorite sites (pretty much what the first iteration of this PR did)